src/algorithm/native/to_wkt.rs

```diff
@@ -376,17 +376,130 @@ fn add_coords<W: Write, C: CoordTrait<T = f64>>(
     writer: &mut W,
     mut coords: impl Iterator<Item = C>,
 ) -> Result<(), Error> {
-    writer.write_str("(")?;
+    writer.write_char('(')?;
 
     let first = coords.next().unwrap();
     add_coord(writer, first)?;
 
     for coord in coords {
-        writer.write_str(",")?;
+        writer.write_char(',')?;
         add_coord(writer, coord)?;
     }
 
-    writer.write_str(")")?;
+    writer.write_char(')')?;
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use arrow_buffer::OffsetBuffer;
+
+    use super::*;
+    use crate::{
+        array::{CoordBuffer, InterleavedCoordBuffer},
+        scalar::{
+            OwnedLineString, OwnedMultiLineString, OwnedMultiPoint, OwnedMultiPolygon, OwnedPoint,
+            OwnedPolygon,
+        },
+    };
+
+    #[test]
+    fn point() {
+        let coords = InterleavedCoordBuffer::<2>::new(vec![1., 2.].into());
+        let point = OwnedPoint::new(CoordBuffer::Interleaved(coords), 0);
```

Comment:
I think it would be easier to use `crate::test::point::point_array` and then access the first point.